tests/e2e/conftest.py

```
@@ -870,6 +874,35 @@ async def hosted_google_oauth_refresh_server(
+            "GITHUB_OAUTH_CLIENT_ID": "hosted-github-client-id",
+            "GITHUB_OAUTH_CLIENT_SECRET": "hosted-github-client-secret",
+        },
+        extra_result={"emulate_github_url": emulate_github_server["url"]},
+    ):
+        yield server
+
+
+@pytest.fixture(scope="session")
+async def hosted_google_oauth_refresh_server(hosted_google_emulate_server):
+    """Compatibility fixture for hosted Gmail OAuth refresh regression tests."""
+    yield hosted_google_emulate_server
```

Ruff (0.15.18)

[warning] No teardown in fixture `hosted_google_oauth_refresh_server`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)